lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -425,28 +422,31 @@ def main():
         target = "{0}/{1}.sql".format(os.getcwd(), db)
         target = os.path.expanduser(target)
 
-    try:
-        pgutils.ensure_libs(sslrootcert=module.params.get('ssl_rootcert'))
-        db_connection = psycopg2.connect(database=maintenance_db, **kw)
+    if state not in ["dump", "restore"]:
+        if not HAS_PSYCOPG2:
```

Comment:
[`pgutils.ensure_libs`](https://github.com/ansible/ansible/blob/34671a64b30a854f741cc87a4ced79d34270a5b2/lib/ansible/module_utils/postgres.py#L42) (which is called below) already checks `psycopg2` availability.

Reply:
I just change the location of the check that was done on https://github.com/ansible/ansible/blob/0f0d33a9549798a27a1763b4cb48a0e52530cda3/lib/ansible/modules/database/postgresql/postgresql_db.py#L388

It felt better not to change it, but I can do it also.

Reply:
Because the `HAS_PSYCOPG2` check is performed in `pgutils.ensure_libs` which is called just below, this one can be removed.